### ISSUE TITLE
⚠️ Not detecting travis-ci.com statuses

### ISSUE THREAD

**Issue by wolfy1339** [greenkeeperio/greenkeeper#944]
TravisCI has recently stopped using the commit `status`, and has switched to only using the `checks` API. 
This breaks greenkeeper, because it'll push the commits, and will wait until CI finishes, but will never do anything because of that change.

Ref: https://blog.travis-ci.com/2018-09-27-deprecating-github-commit-status-api-for-github-apps-managed-repositories

**Comment by Realtin** [comment]
Hey, did you update the permissions for greenkeeper? The status checks require a new permission from the user otherwise greenkeeper is not allowed to receive the webhooks. It might be hard to find, there should be a highlighted section in the settings for greenkeeper in github

**Comment by wolfy1339** [comment]
I haven't seen that greenkeeper was requesting new permissions. Usually if an app wants new permissions I get an email notification.

I just checked, and greenkeeper does have access to the checks API
________________________________

**Comment by wolfy1339** [comment]
Is it an error that this was closed?

Greenkeeper does have access to checks. I have double checked.
________________________________

**Comment by Realtin** [comment]
oh ok, sorry I read that as the missing permissions were the cause

**Comment by travi** [comment]
if its helpful, my projects continue to get the PRs from greenkeeper correctly. we use travis for public and private projects and have not seen any problem with the PRs getting opened

**Comment by Realtin** [comment]
@wolfy1339 can you write me the affected repositories, here or via support@greenkeeper.io.
I couldn't find something in the logs so far.

**Comment by wolfy1339** [comment]
https://github.com/hellomouse/GNS
It created the branch and all, but didn't create a PR. I created the PR manually myself

**Comment by wolfy1339** [comment]
Greenkeeper just created another branch with no PR. Here's the specific branch: https://github.com/hellomouse/GNS/tree/greenkeeper/%40octokit/rest-15.13.1

**Comment by travi** [comment]
you have that dependency defined as a range. greenkeeper creates a branch to determine if in-range updates break your build, but doesn't open a PR for in-range updates because your defined range would already include them. you will only get PRs for updates that fall outside your defined range.

https://greenkeeper.io/docs#greenkeeper-step-by-step

**Comment by wolfy1339** [comment]
Alright, but then the branch shouldn’t be left there afterwards? Greenkeeoer should clean up after itself like it does when it’s PRs are merged

**Comment by travi** [comment]
yeah, that should be what happens. if that's not the case, might be worth changing the title of this issue to make it clear what the problem really is
